charts/ceph-csi-rbd/values.yaml

```diff
@@ -31,9 +31,15 @@ csiConfig: []
 # Ref: https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md
 # Example:
 # encryptionKMSConfig:
-#   - encryptionKMSID: "<kms-id>"
-#     <kms-specific-configs>
-encryptionKMSConfig: []
+#   vault-unique-id-1:
```

Comment:
@vasyl-purchel can we add some dcoumentation around this "unique" id too ?

Reply:
it's already documented in deploy-rbd.md:
> To use Vault as KMS set `encryptionKMSID` to a unique identifier for Vault
configuration. You will also need to create vault configuration similar to the
[example](../examples/rbd/kms-config.yaml) and use same `encryptionKMSID`.